src/collar/oc_bookmarks.lsl

```diff
@@ -120,11 +118,15 @@ UserCommand(integer iNum, string sStr, key kID) {
     } else if (sStr == "rm bookmarks") {
         if (kID!=g_kWearer && iNum!=CMD_OWNER) llMessageLinked(LINK_DIALOG,NOTIFY,"0"+"%NOACCESS%",kID);
         else Dialog(kID,"\nDo you really want to uninstall the "+g_sSubMenu+" App?", ["Yes","No","Cancel"], [], 0, iNum,"rmbookmarks");
-    } else if(sStr == PLUGIN_CHAT_CMD || llToLower(sStr) == "menu " + PLUGIN_CHAT_CMD_ALT || llToLower(sStr) == PLUGIN_CHAT_CMD_ALT) {
+    }else if(sStr == PLUGIN_CHAT_CMD || llToLower(sStr) == "menu " + PLUGIN_CHAT_CMD_ALT || llToLower(sStr) == PLUGIN_CHAT_CMD_ALT) {
+        if (iNum==CMD_GROUP)
+            llMessageLinked(LINK_DIALOG,NOTIFY,"0"+"%NOACCESS%",kID);
         // an authorized user requested the plugin menu by typing the menus chat command
         DoMenu(kID, iNum);
-    } else if(llGetSubString(sStr, 0, llStringLength(PLUGIN_CHAT_CMD + " save") - 1) == PLUGIN_CHAT_CMD + " save") {
-        //grab partial string match to capture destination name
+    } else if(llGetSubString(sStr, 0, llStringLength(PLUGIN_CHAT_CMD + " save") - 1) == PLUGIN_CHAT_CMD + " save") {           if (iNum==CMD_GROUP){
```

Comment:
Is there an accidentally dropped newline here?

Reply:
umm yes never changed this

Reply:
If you look at the end of your line 126 (alllll the way to the right), there's a "if (iNum==CMD_GROUP){" hanging out, that I don't see in the previous version.

Reply:
Yes my bad sorry, most scripts were largely untouched except for the section of the top.
Dropped newlines etc may have happened during copy pasting between grids as code was copied from SL -> Inworlds (cleaned working there) -> Private OpenSim 0.9 Grid (cleaned up working) -> copied to TAG who run a slightly different version again to lsl interpreter (and copied working before coping to github for pull request.